### DISCUSSION TITLE
Can't login into the linux app

### DISCUSSION THREAD

**Discussion by zenzawa** [ente-io/ente#3078] [Q&A]
First I entered the password but it said that my device is not powerful enough. Idk how is that possible cause I have a ryzen 7 + 4060. Then I tried using recovery keys but it said I entered incorrect one even though I just copy pasted it from my mobile phone's ente auth app

**Comment by EmilEmilchen** [comment]
Same problem here. Did you ever find a solution?

**Comment by EmilEmilchen** [comment]
Nevermind, see #2325
